fineract-provider/src/main/java/org/apache/fineract/portfolio/self/security/api/SelfUserApiResource.java

```diff
@@ -59,7 +62,12 @@ public SelfUserApiResource(final UsersApiResource usersApiResource,
         }
 
         @PUT
-        public String update(final String apiRequestBodyAsJson) {
+        @Consumes({ MediaType.APPLICATION_JSON })
```

Comment:
why have you added this line?
not in original code!

Reply:
@Sanyam96 Please note that I have added these lines because it is consuming and producing JSON.

Reply:
we don't have to put these annotations in the code as they don't belong to Swagger

Reply:
okay... removing